### PR TITLE
[CLNP-2908] Typescript error fix for hooks

### PR DESCRIPTION
**Pull request by chrisallo**
Fixes: [CLNP-2908](https://sendbird.atlassian.net/browse/CLNP-2908)

[CLNP-2908]: https://sendbird.atlassian.net/browse/CLNP-2908?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -105,6 +105,7 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^14.4.3",
+    "@types/node": "^20.12.7",
```
Comment (bang9) [inline package.json:108]: Q. Why do we need @types/node

File: src/hooks/VoiceRecorder/useVoiceRecorder.tsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ export const useVoiceRecorder = ({
 
   // Timer
   const [recordingTime, setRecordingTime] = useState<number>(0);
-  let timer: ReturnType<typeof setInterval> = null;
+  let timer: ReturnType<typeof setInterval> | null = null;
```
Comment (bang9) [inline src/hooks/VoiceRecorder/useVoiceRecorder.tsx:76]: This isn't related to this PR, but it seems like this code should be updated with `useRef`
  
```ts
const timer = useRef<ReturnType<typeof setInterval> | null>(null);

// ...
timer.current = setInterval;

// ...
clearInterval(timer.current);
timer.current = null;
```

File: package.json (hunk 1 above)
Comment (chrisallo) [reply package.json:102]: Cannot remind what has happened with this change. Will revert it for now.

File: src/hooks/VoicePlayer/useVoicePlayer.tsx [hunk 3]
```diff
@@ -25,10 +25,10 @@ export interface UseVoicePlayerContext {
 export const useVoicePlayer = ({
   key = '',
   channelUrl = '',
-  audioFile = null,
+  audioFile,
   audioFileUrl = '',
 }: UseVoicePlayerProps): UseVoicePlayerContext => {
-  const [groupKey] = useState<string>(generateGroupKey(channelUrl, key));
+  const groupKey = generateGroupKey(channelUrl, key);
```
Comment (bang9) [inline src/hooks/VoicePlayer/useVoicePlayer.tsx:31]: note: state 로 관리하면 url,key 업데이트를 해주는 로직이 필요, 일반 string 사용하도록 수정

File: src/modules/ChannelSettings/components/ModerationPanel/MembersModal.tsx
Comment (bang9) [inline src/modules/ChannelSettings/components/ModerationPanel/MembersModal.tsx:1]: 👍

**Review by bang9 — APPROVED** [review]

**Pull request merged**